### DISCUSSION TITLE
如何在编辑器中隐藏部分富文本

### DISCUSSION THREAD

**Discussion by DeveloperHu** [wangeditor-next/wangEditor-next#451] [General]
## 问题描述

*请输入遇到的问题...*

## wangEditor 版本

*请输入内容……*

## 是否查阅了文档 ？

（文档链接 [https://cycleccc.github.io/docs/](https://cycleccc.github.io/docs/) ）

*是/否*

## 最小成本的复现步骤

（请告诉我们，如何**最快的**复现该问题？）

- 步骤一
- 步骤二
- 步骤三

**Comment by DeveloperHu** [comment]
直接在gethtml中获取的内容加上style display none也无效

**Comment by cycleccc** [comment]
没有支持 display: none 这个属性，自定义元素试试？好奇怪的需求，在富文本里面隐藏部分文本。
方便说下为什么有这个需求吗？

**Comment by DeveloperHu** [comment]
我们产品用富文本编辑邮件脚本有一些脚本代码不想在富文本前端使用，传输到后端时需要存在。怎么让style支持display呢大佬

**Comment by cycleccc** [comment]
这种推荐使用自定义元素，具体的可以看文档、源码中的自定义元素插件以及网上别人写的用例

**Comment by cycleccc** [comment]
隐藏编辑内容建议使用自定义元素实现，不支持直接以 display:none 作为内置样式。本讨论现关闭。
